packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.js

```diff
@@ -32,9 +33,12 @@ const styles = theme => ({
  */
 function BreadcrumbCollapsed(props) {
   const { classes, ...other } = props;
+
   return (
-    <li className={classes.root} {...other}>
-      <MoreHorizIcon className={classes.icon} />
+    <li className={classes.root}>
+      <ButtonBase role="button" focusRipple {...other}>
```

Comment:
I wonder if we couldn't merge the ButtonBase and the li element.
I doubt we need the role button.